.github/workflows/_buildpacks-release.yml

```diff
@@ -147,7 +147,7 @@ jobs:
 
   publish-docker:
     name: Publish → Docker - ${{ matrix.buildpack_id }}
-    needs: [compile]
+    needs: [ compile ]
```

Comment:
Could you revert the whitespace formatting changes to the `needs` attributes?

The GitHub Actions workflow reference doesn't use whitespace padding style for these:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds

And neither does https://www.yaml.info/learn/bestpractices.html nor Prettier in YAML mode.

Reply:
Hmm, must be a new formatting setting on RustRover. I don't remember my IDE adding those before. Anyways, I disabled that extra formatting and saved with the whitespace removed.